src/reducers/inputs.js

```diff
@@ -85,6 +85,10 @@ export const setMySelections = (state, action) => {
       obj[key].items = action.payload.items
     }
 
+    if (!Object.keys(obj).includes(key)) {
```

Comment:
I think we are duplicating this code in the `setMyItems` in this reducer https://github.com/LD4P/sinopia_editor/blob/890b116ebb3f195d77969bd22e04abe2e488fd2a/src/reducers/inputs.js#L29-L44, 
Maybe collapse these functions or create a new function that both `setMySelections` and `setMyItems` call?

Reply:
@jgreben, should I can make this a separate issue for follow-up work?

Reply:
> @jgreben, should I can make this a separate issue for follow-up work?

No need, I will take a look and make a follow-up commit.

Reply:
@jermnelson I combined the `setMyItems` and `changeMySelections` reducers into a single `setItemsOrSelections` reducer. LMK if this is what you had in mind.

Reply:
LGTM; Thanks for doing this!